### ISSUE TITLE
getInsertRevisionSQL with schema

### ISSUE THREAD

**Issue by acogent** [simplethings/EntityAudit#133]
Hello,

Sorry for my english !

I use schema in my database.
With doctrine 2.4, it was ok.
But now, when i want to update something, i have an error. 

I modify getInsertRevisionSQL like this : 
```
$table_    = $class->table;
$prefix    = '';

 if (isset( $table_['schema']) === true) {
        $prefix = $table_['schema'].'.';
}

$tableName    = $this->config->getTablePrefix() . $prefix.$class->table['name'] . $this->config->getTableSuffix();
```
My composer.json
```
"symfony/symfony": "2.6.*",
"doctrine/orm": "~2.2,>=2.2.3",
"doctrine/doctrine-bundle": "~1.2",
```

**Comment by andrewtch** [comment]
Can you copy-paste the error please?

**Comment by acogent** [comment]
The error :

An exception occurred while executing 'INSERT INTO nivf_rn_trg_audit (rev, revtype, rn_cid, rig_alti_cid, trg_date, hauteur_ellips, hauteur_ellips_precision, src_cod_lph, rem_portage, id, trg_cid, altitude, altitude_validite, maj_date) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["16", "UPD", 526675, 2, "1978", null, null, null, "- eeezz", 1, "1000011978100101", "217.092", 1, "2015-06-01 16:25:58"]:

.....


[3/3] TableNotFoundException: An exception occurred while executing 'INSERT INTO nivf_rn_trg_audit (rev, revtype, rn_cid, rig_alti_cid, trg_date, hauteur_ellips, hauteur_ellips_precision, src_cod_lph, rem_portage, id, trg_cid, altitude, altitude_validite, maj_date) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)' with params ["16", "UPD", 526675, 2, "1978", null, null, null, "- eeezz", 1, "1000011978100101", "217.092", 1, "2015-06-01 16:25:58"]:

SQLSTATE[42P01]: Undefined table: 7 ERREUR: la relation « nivf_rn_trg_audit » n'existe pas
LINE 1: INSERT INTO nivf_rn_trg_audit (rev, revtype, rn_cid, rig_alt...


...

[2/3] PDOException: SQLSTATE[42P01]: Undefined table: 7 ERREUR: la relation « nivf_rn_trg_audit » n'existe pas
LINE 1: INSERT INTO nivf_rn_trg_audit (rev, revtype, rn_cid, rig_alt...
^   +

**Comment by UFTimmy** [comment]
I am having the same issue, so hopefully I can try to explain it better. Basically, this bundle doesn't handle Entities which are configured with a schema, at least in newer versions of Doctrine.

I use MSSQL, so each table is in a schema. dbo is the default schema, but all of our tables are in actual schemas. Doctrine handles this, it allows you to name tables with two parts (schemaName.tableName). In previous versions of Doctrine, the schema and table name were stored together, so if you asked for the table name, you got the schema too. In newer versions, when you get the table name from Doctrine, you only get the table name. You have to ask for the schema name.

I've attached an Xdebug screen shot. This is a blocking issue for me, I cannot use the bundle because the bundle cannot select any data from the audit tables. In summary, my table is oasis.InstructionMode. The audit table was successfully built as oasis.InstructionMode_audit. However, when the Reader is trying to pull data out of it, it's trying to pull data out of just InstructionMode_audit, without the schema.

![image](https://cloud.githubusercontent.com/assets/7061770/8456082/0cce868e-1fd7-11e5-84be-c582fd3b31a4.png)

**Comment by UFTimmy** [comment]
If this is helpful, this is the Doctrine ORM commit that made the change: https://github.com/doctrine/doctrine2/commit/51bf82b7e75e677916faf08fa1afa19590e35f93

Prior to that, entities with a schema kept the schema in the name key. After that, it was split up and you have to request the schema separately. I believe this was tagged in doctrine ORM 2.5

**Comment by DavidBadura** [comment]
this should be solved with #155
